docs/asyncapi.yaml

```diff
@@ -90,3 +93,27 @@ components:
       type: string
       description: PDF filename
       example: FFC_PaymentStatement_SFI_2022_1234567890_2022080515301012.pdf
+    Scheme:
+      type: object
+      required:
+      - name
+      - shortName
+      - year
+      - frequency
+      properties:
+        name:
+          type: string
+          description: The name of the scheme
+          example: Sustainable Farming Incentive
+        shortName:
+          type: string
+          description: The short name of the scheme
+          example: SFI
+        year:
+          type: string
```

Comment:
this is going to influence the mapping in the constructor -- do you want everything as string?

Reply:
Pretty much everything that comes out of the constructor is a string as downstream are just presentation services. (for now)  The `generator` repo will reject a non-string for this before it makes it this far.